.github/workflows/if-nodejs-release.yml

```diff
@@ -80,6 +80,12 @@ jobs:
       - if: steps.packagejson.outputs.exists == 'true'
         name: Install dependencies
         run: npm install
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+        with:
+          platforms: 'linux/amd64,linux/arm64'
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
```

Comment:
I cannot accept such a changes. Almost all workflows in the `.github/workflows` folder are copied (by our one global workflow) from this place - https://github.com/asyncapi/.github/tree/master/.github/workflows, so it means that if someone will update workflows in `.github` repo, updated workflows in this PR will be overwritten.

If you need to add such a setup in workflow, you should add these steps in appropriate workflows in the https://github.com/asyncapi/.github/tree/master/.github/workflows. Update `generate:docker` script is ok, because every repo can perform different logic for assets generation.

cc @derberg What do you think? Should we move these steps to the `.github` repo? I ask because I know that you created PR with some updates on that workflows related to the automatic release and also that PR made changes related to docker release. How you think, how we should do this? @sandoval Could you wait for @derberg answer?

Reply:
@sandoval as @magicmatatjahu wrote, whatever you change here, will anyway be overwritten (have a look at the comment at the beginning of the workflow.

yeah, so at the moment, I'm rebuilding a bit the release pipeline to have it less error-prone. The idea is to handle docker release after main release, like we do it here https://github.com/asyncapi/github-action-for-generator/blob/master/.github/workflows/release-docker.yml (example ob publishing docker after GitHub release)

so @magicmatatjahu @sandoval, this is what I suggest you do, really. Have a separate workflow that handles docker release.  Design it as generic as possible, call file `if-docker-release.yml`. Then once we add it in the `.github` repo, we will just override and long-term maintain from there. I hope is makes sense what I'm writing 🤔

Reply:
So as I understand @derberg, that new workflow should be created in this PR, and then we should move it to the `.github` repo, right? Or should we start from creating workflow in the `.github` repo.

Reply:
I definitely suggest doing it here first and having it battle-tested here, we can move it over after a release or 2 to `.github` repo alone, or @sandoval can do it if he is interested

Reply:
Thank you for the comments @magicmatatjahu and @derberg ! I also think it would be best to have a separate workflow for building the docker image, based on the github actions for building such images. However, I tried to adapt to the project's logic when submitting the PR.
In using the github actions, I would use [docker-build-push](https://github.com/docker/build-push-action) action instead of `semantic-release`, what do you think?

Reply:
@sandoval It will be problematic to run in two independent jobs `semantic-release`, so yeah, `[docker-build-push](https://github.com/docker/build-push-action)` should be better. However, next to `latest` version we should also push version indicated by `package.json`.

Reply:
yup, forget semantic-release

as you will react to release event, you will not have access yet to updated package.json with proper version. just look at the action I linked few comments earlier. there is an example how I get package version from release tag, just copy it